test/DynamoCoreWpfTests/WorkspaceGeneralOperations.cs

```diff
@@ -113,7 +113,64 @@ public void VerifyGlobalLacingStrategyUpdatesForLacingStrategies()
             AssertPreviewCount(nodeIds[1].ToString(), 2);
             AssertPreviewCount(nodeIds[2].ToString(), 2);
         }
+        
+        [Test]
+        public void VerifyLacingStrategyForCodeBlockNodes()
```

Comment:
We usually follow `Arrange-Act-Assert` pattern to structure tests, it makes them a bit easier to read, can we arrange everything according to that here? 

heres an example of other tests we did:
https://github.com/DynamoDS/Dynamo/pull/10607/files#diff-95c6b0530d0c2849727a5848fc9b60950b26584aad6f1e8759308af0b0abccae

and more on AAA
https://docs.microsoft.com/en-us/visualstudio/test/unit-test-basics?view=vs-2019#write-your-tests